src/Symfony/Bridge/PhpUnit/Tests/ClockMockTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpUnit\Tests;
+
+use Symfony\Bridge\PhpUnit\ClockMock;
+
+class ClockMockTest extends \PHPUnit_Framework_TestCase
+{
+    public function testTimeNowNull()
```

Comment:
this method tests only if `ClockMock::time()` returns int value. 
Maybe try to focus more on the class behaviour rather than code coverage?. Method names are not descriptive enough IMO. `testTimeNowNull` doesn't tell me you want to `test_if_time_returns_int_when_not_modified`.

How about testing if `time()` returns equal or greater value than previously run php `time()` function and less or equal than the same function run afterwards?